src/tools/rbd/action/Bench.cc

```diff
@@ -16,13 +16,16 @@
 
 namespace rbd {
 namespace action {
-namespace bench_write {
+namespace bench{
 
 namespace at = argument_types;
 namespace po = boost::program_options;
 
 namespace {
 
+#define BENCH_READ	true
+#define BENCH_WRITE	false
+
```

Comment:
Won't we want in future another IO types, e.g read/write mix? May be use enum instead of bool?